dataprofiler/tests/profilers/profiler_options/test_profiler_options.py

```diff
@@ -28,17 +28,20 @@ def test_default_profiler_options(self, *mocks):
         for column in profile.options.properties:
             self.assertTrue(profile.options.properties[column].is_enabled)
 
-        for column in ["int", "float", "text"]:
-            column = profile.options.properties[column]
+        for column_type in ["int", "float", "text"]:
+            column = profile.options.properties[column_type]
             self.assertTrue(column.properties["histogram_and_quantiles"])
             self.assertTrue(column.properties["min"])
             self.assertTrue(column.properties["max"])
             self.assertTrue(column.properties["sum"])
             self.assertTrue(column.properties["variance"])
             self.assertTrue(column.properties["is_numeric_stats_enabled"])
-            self.assertFalse(column.properties["num_zeros"].is_enabled)
-            self.assertFalse(column.properties["num_negatives"].is_enabled)
-
+            if(column_type is not "text"):
```

Comment:
Should do `column_type != "text"`

Reply:
also can remove the `()` around the statements